diff --git a/mall-mbg/src/main/java/com/macro/mall/model/PmsProductReview.java b/mall-mbg/src/main/java/com/macro/mall/model/PmsProductReview.java
index 9d88ea1..f14f467 100644
--- a/mall-mbg/src/main/java/com/macro/mall/model/PmsProductReview.java
+++ b/mall-mbg/src/main/java/com/macro/mall/model/PmsProductReview.java
@@ -1,29 +1,25 @@
 package com.macro.mall.model;
 
-import com.baomidou.mybatisplus.annotation.FieldFill;
-import com.baomidou.mybatisplus.annotation.IdType;
-import com.baomidou.mybatisplus.annotation.TableField;
-import com.baomidou.mybatisplus.annotation.TableId;
-import com.baomidou.mybatisplus.annotation.TableName;
+import com.baomidou.mybatisplus.annotation.*;
 import lombok.Data;
-
-import java.util.Date;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
 
 @Data
 @TableName("pms_product_review")
 public class PmsProductReview {
-    
+
     @TableId(type = IdType.AUTO)
     private Long id;
-    
+
     private Long productId;
-    
+
     private Long memberId;
-    
+
     private Integer star;
-    
+
     private String content;
-    
+
     @TableField(fill = FieldFill.INSERT)
-    private Date createTime;
-}
+    private LocalDateTime createTime;
+}
\ No newline at end of file
